### PR TITLE
WIP: Send webhook event with junit report

### PR DESCRIPTION
**Pull request by jjusianiec**
Sample HTTP event
```
{
   "owner":"allegro-internal",
   "repo":"akar-test-app",
   "sha":"247dfb05345d11a3c5f5ab3331bef36d77a36811",
   "checkRun":{
      "id":2573603741,
      "nodeId":"MDg6Q2hlY2tSdW4yNTczNjAzNzQx",
      "checkSuiteId":2732049540
   },
   "ref":"refs/heads/CERBER-658",
   "action":"allegro-actionsjunit-reporter",
   "runNumber":63,
   "runId":838131533,
   "testResults":[
      {
         "name":"should respond with 'local resource' when asking for local resources",
         "classname":"pl.allegro.tech.dummy.akartestapp.ExampleIntegrationSpec",
         "result":"successful"
      },
      {
         "name":"should fail",
         "classname":"pl.allegro.tech.dummy.akartestapp.ExampleIntegrationSpec",
         "result":"failed"
      },
      {
         "name":"should skip",
         "classname":"pl.allegro.tech.dummy.akartestapp.ExampleIntegrationSpec",
         "result":"skipped"
      },
      {
         "name":"outside in test",
         "classname":"pl.allegro.tech.dummy.akartestapp.ExampleMvcSpec",
         "result":"successful"
      },
      {
         "name":"should construct domain object (its just an example, never write tests like this!)",
         "classname":"pl.allegro.tech.dummy.akartestapp.domain.DomainObjectSpec",
         "result":"successful"
      }
   ]
}
```

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -16,6 +16,9 @@ inputs:
     description: 'Matrix context of job'
     required: false
     default: ${{ toJson(matrix) }}
+  webhook-url:
+    description: 'Specifies the endpoint URL where the JUnit test results will be send'
```
Comment (deejay1) [inline action.yml:20]: `will be sent`

File: src/event.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import Report from './Report';
+import * as github from '@actions/github';
+import * as core from '@actions/core';
+import axios from 'axios';
+import {TestCase} from 'junit2json';
+import {CheckRun} from './check';
+
+export async function send(report: Report, checkRun: CheckRun): Promise<void> {
+    const url = core.getInput('webhook-url');
+    core.info(JSON.stringify(process.env, null, 2));
```
Comment (deejay1) [inline src/event.ts:10]: won't this potentially leak credentials ?

File: src/event.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        core.info(`Sending test results [${testResults.length}] to webhook endpoint.`);
+        await axios.post(url, {
+            ...github.context.repo,
+            sha: github.context.sha,
+            checkRun: checkRun,
+            ref: github.context.ref,
+            action: github.context.action,
+            runNumber: github.context.runNumber,
+            runId: github.context.runId,
+            testResults: testResults,
```
Comment (deejay1) [inline src/event.ts:41]: we should probably get some authentication in here

File: src/event.ts (hunk 2 above)
Comment (jjusianiec) [reply src/event.ts:10]: thx for comments, I'm still working on it. I'm closing PR for now.

**Pull request closed without merging**